kong/core/reports.lua

```diff
@@ -180,6 +180,9 @@ return {
 
     create_timer(PING_INTERVAL, ping_handler)
   end,
+  rewrite = function()
+    -- Do nothing
+  end,
```

Comment:
I don't think we need this? We manually call `reports.init_worker()` and `reports.log()`, but I see nowhere where `reports.rewrite()` is called. There is no `reports.access()` either, for example and for that matters.